packages/mdc-select/bottom-line/constants.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const cssClasses = {
+  BOTTOM_LINE_ACTIVE: 'mdc-select__bottom-line--active',
```

Comment:
README shows this as `mdc-select-bottom-line--active`

Reply:
I would be inclined to think that `mdc-select__bottom-line` actually makes more sense though, at least while its bottom line implementation is still within this package...

Reply:
will use `mdc-select__bottom-line`